main.go

```diff
@@ -286,6 +212,37 @@ func findDeployPth(deployDir, baseName, ext string) (string, error) {
 	return deployPth, err
 }
 
+func validateAndMigrateConfig(config *Config) error {
+	if config.GradleFile != "" {
+		if exist, err := pathutil.IsPathExists(config.GradleFile); err != nil {
+			return fmt.Errorf("failed to check if GradleFile exists at: %s, error: %s", config.GradleFile, err)
+		} else if !exist {
+			return fmt.Errorf("GradleFile does not exist at: %s", config.GradleFile)
+		}
+	}
+
+	if config.ApkFileIncludeFilter != "" {
+		log.Warnf("Please use *APK and AAB file include filter* instead of *APK file include filter*.")
```

Comment:
i would put more pressure on the user, i used something like this: `step input 'APK file path' (apk_path) is deprecated and will be removed soon, use 'APK or App Bundle file path' (app_path) instead!`

Reply:
Other than that, if the user already has a custom filter and the new filter is on default we should use the old filter, to do not break existing configs

Reply:
It should behave like that.

Reply:
the default include filter is `*.apk`, if the user defined something else, like `*debug*.apk` and the new filter is on default value `*.app\n*.aab`, then use the old filter's value.